modules/sentinel-watchlist-item/vars.tf

```diff
@@ -0,0 +1,15 @@
+variable "name" {
+  description = "The name in UUID format which should be used for this Sentinel Watchlist Item."
+  type        = string
+  default     = null
```

Comment:
Should this default to null? 

Shouldn't this be a required var?

Reply:
Its null as default, just to maintain consistency with the terraform provider, where name is an optional value
https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/sentinel_watchlist_item#name